### PR TITLE
Improve service page

### PR DESCRIPTION
**Pull request by tengqm**
This amends the service page for LoadBalancer type of services.

### CODE REVIEW

File: content/en/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -1100,21 +1103,15 @@ about the API object at: [Service API object](/docs/reference/generated/kubernet
...
 ### UDP
 
-{{< feature-state for_k8s_version="v1.0" state="stable" >}}
-
 You can use UDP for most Services. For type=LoadBalancer Services, UDP support
 depends on the cloud provider offering this facility.
 
 ### HTTP
 
-{{< feature-state for_k8s_version="v1.1" state="stable" >}}
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/service.md]: If dropping these, there is also a feature-state shortcode for PROXY protocol. Maybe drop that as well?

Comment (tengqm) [reply content/en/docs/concepts/services-networking/service.md]: Good spot. Fixed.

File: content/en/docs/concepts/services-networking/service.md [hunk 2]
```diff
@@ -548,6 +548,9 @@ status:
 
 Traffic from the external load balancer is directed at the backend Pods. The cloud provider decides how it is load balanced.
 
+For LoadBalancer type of Services, when there are more than one port defined, all
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/service.md]: Actually another tiny nit
```suggestion
For LoadBalancer type of Services, when there is more than one port defined, all
```

(English is weird!)

Comment (tengqm) [reply content/en/docs/concepts/services-networking/service.md]: Got it. Thanks.

**Comment by tengqm** [comment]
Guard against multi-protocols for LB services: https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/validation/validation.go#L4001
Guard against protocols other than TCP, UDP or SCTP:
https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/validation/validation.go#L4006

**Pull request merged**